README.md

```diff
@@ -111,6 +108,11 @@ The quickstart backend is now running on http://localhost:8000 and frontend on h
 If you make changes to one of the server files such as `index.js`, `server.go`, etc, or to the
 `.env` file, simply run `make up language=node` again to rebuild and restart the container.
 
+If you experience a Docker connection error when running the command above, try the following:
+
+- Make sure Docker is running
+- Try running the command prefixed with `sudo`
```

Comment:
this _should_ never be needed, as in if you need to do this you likely set up your development environment or docker incorrectly, but i guess its ok to call out the corner case